cfgov/tccp/enums.py

```diff
@@ -44,13 +44,27 @@ def make_choices(*choices):
 ]
 
 
+CreditTierConciseChoices = [
```

Comment:
Would it make sense to combine these with `CreditTierColumns` since the tier label from the dataset is repeated in both? Like:

```python
CreditTiers = {
    "tier_0": {
        "dataset_label": "No credit score",
        "concise_label": "No score",
        "column_label": "none"
    },
    "tier_1": {
        "dataset_label": "Credit score 619 or less",
        "concise_label": "619 or less",
        "column_label": "poor"
    },
    …
}
```

Might be overkill, especially since the "no score"  tier isn't used for APR calculations. Totally up to you.

Reply:
Yes, the items in this file could definitely be consolidated to avoid duplication. The original idea behind this file was to enumerate the possible values of fields in the TCCP dataset. But we also want to use these as Django [choices](https://docs.djangoproject.com/en/5.0/ref/models/fields/#choices) for form fields, [for example](https://github.com/cfpb/consumerfinance.gov/blob/4700bb420d52757847ef3f8cd238eca3622cc55a/cfgov/tccp/filterset.py#L18). I'm inclined to leave as-is to keep that usage straightforward even with the duplication.